### PR TITLE
[NONMODULAR] Makes energy guns, and laser guns for the most part normal sized.

### PR DESCRIPTION
**Pull request by RatFromTheJungle**
## About The Pull Request

What it says on the tin.
## How This Contributes To The Skyrat Roleplay Experience

A solid majority of our ballistics are already normal sized, while having magazines, and having energy guns both be hard to 'reload,' and equally hard to carry means they're placed into a box of impracticality, which making them easier to carry should even up on.

Also the disabler SMG is normalsized, despite being better then the L-carbine(bulky) for some ungodly reason

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
2do2morrow
</details>

## Changelog

:cl:
balance: Energy/laser guns are now normalsized, and thus, backpack/secbeltable
/:cl:

### CODE REVIEW

**Comment by nikothedude** [comment]
why not just make the ballistics bulky instead of powercreeping

**Comment by RatFromTheJungle** [comment]
> why not just make the ballistics bulky instead of powercreeping

because that's much less fun, engaging, or grounding to have a bunch of pistols that are inexplicably bulky sized for the sake of making energy guns seem appealing

**Comment by projectkepler-ru** [comment]
@MrMelbert 
post that image

**Comment by MrMelbert** [comment]
Those Who Do Not Learn History Are Doomed To Repeat It

image
image

**Comment by nikothedude** [comment]
> > why not just make the ballistics bulky instead of powercreeping
> 
> because that's much less fun, engaging, or grounding to have a bunch of pistols that are inexplicably bulky sized for the sake of making energy guns seem appealling

on the contrary, it already makes no sense as to why half the guns are bulky
this is for balance, the laser guns were made bulky for balance, if the issue is that everyone is holding like 10 bagillion pistols we have two options: make the pistols hard to get or make them hard to store/limit their ammo, powercreeping the rest of the guns is always a bad idea

**Comment by Namwty** [comment]
You can almost put most smgs and pistols ( extended mags 24 round btw, remember ) in the bags, while you can't put lasers, even tho they have less rounds and generally are easy to counter.


Ballistics have good armor pen + they have armor pen rounds, which basically makes hard to counter full stop. The only real good one combination wise would be tough from DNA vault ( 30% dmg reduction or something for brute ). Even if someone stacks laser guns in their bag, it won't be enough rounds to counter 6 extended mags that you have in your bag.

Lasers are easier to counter compared to ballistics, reflector type armor shits on lasers compared to our bellowed bulletproof vests, which barely can even handle non armor piercing rounds. Fiery sweat from genetics provides 50% burn protection full stop ( way easier to obtain than DNA tough and easier than hulk or anything in those lines ). **Going to mention that one EMP will effect basically every damn gun you have in your bag.** Arguably only thing that lasers are truly good for is windows or fighting races like snails who take like 2x dmg from burns.

Also bullet visibility, lasers are more noticeable and easier to dodge, especially during lag ( not me with high ping ass tho ). 

Most people don't even bother to wear anything other than GZ or bulletproof gear/riot for example ( has 1 laser ) cause literally MOST people are using ballistics on the server, it's so rare to see that it's not even a concern. Reflector trenchcoat has probs six layers of dust, cause no one ever needs it.

Example: Bulletproof armor is basically best armor for anti ballistics ( This is NOT AP rounds )

https://github.com/Skyrat-SS13/Skyrat-tg/assets/104979184/587dbecd-c6f9-493f-9ab6-332cde625eed

Example: Reflector + Fiery sweat lmao vs HELLFIRE lasers

https://github.com/Skyrat-SS13/Skyrat-tg/assets/104979184/ceea0a1c-a535-4ac1-9454-b627d3dc7a5c

Example: Normal helmet ONLY vs SMG non AP

https://github.com/Skyrat-SS13/Skyrat-tg/assets/104979184/72bd8042-c299-468f-bd8d-3b65256f0f1a

Example: Normal Helmet ONLY + Fiery sweat vs laser carbine

https://github.com/Skyrat-SS13/Skyrat-tg/assets/104979184/6069f0e5-845e-4909-be49-fb07a0f2053d



Not even going to bother bringing up heavier stuff like machineguns from cargo or fact that people tend to dodge bullets, and at that rate higher ammo count matters.

And regarding Fiery sweat igniting you, the rate of ignition and fire stacks depend on **how much instability you have**... fiery sweat has 0 instability even if you use mutator, having 0 instability, you are looking at being ignited once 30~ mins ( if not unlucky as hell ) or longer with like 1 fire stack that can be rolled within 2 seconds ( Not going to bring chromosomes that can reduce rate by 50% more, which takes like minute to obtain anyhow ).

TL;DR
Go to genetics and ask for Fiery sweat then laugh at laser users emptying their clip into you, just to see you standing.
Reflector armor >>>>>>> bulletproof armor
Lasers are easy to counter, compared to ballistics and probs deserve a bit love.

Alternatively, nerf ballistics or make them rarer.

Regardless of what, we got rules that prevent powergaming, so people who carry 5 guns, be it lasers or ballistics, most likely get slapped by staff.

**Comment by projectkepler-ru** [comment]
I know what I'm gonna do next time I get traitor Rd on bubberstation now ty

**Comment by RatFromTheJungle** [comment]
> > > why not just make the ballistics bulky instead of powercreeping
> > 
> > 
> > because that's much less fun, engaging, or grounding to have a bunch of pistols that are inexplicably bulky sized for the sake of making energy guns seem appealling
> 
> on the contrary, it already makes no sense as to why half the guns are bulky this is for balance, the laser guns were made bulky for balance, if the issue is that everyone is holding like 10 bagillion pistols we have two options: make the pistols hard to get or make them hard to store/limit their ammo, powercreeping the rest of the guns is always a bad idea

not even commenting on the much bigger post above mine, because I can sum this up much short with the reply of, "You can already hold like ten disablers if you're dedicated too it, and once they're in stamcrit, they're good as dead anyway" and a remark akin too, "laser guns aren't even that good, compared to just sawing off shotties" and while servers like TG/whatever have had routine issues with it, we have more validhunting done on-average with stunbatons(which fit in backpacks) then we have with the actual LMGs. 
I doubt lasers are going to cause any issues, while contrastingly, I think 'limit the pistols too!' is just a boring take, and other servers can appeal to that, but I don't really feel like doing so would be in our best interests, if people want pistols they shouldn't have to go through chutes and ladders for the sake of pandering to some non-existent balance demon.

especially one we aren't currently really seeing the ramifications of, as I don't have any examples that come to mind for people carrying a bunch of pistols, because basic headmath adds up to 'there's no advantage to doing that'

**Comment by projectkepler-ru** [comment]
You people do bring up a good points but isn't this just more powercreep?

**Comment by RatFromTheJungle** [comment]
> You people do bring up a good points but isn't this just more powercreep?

depends on definition, but powercreep usually ends up at 'new things makes everything old useless' whereas this is 'makes old thing comparable to new thing, instead of 'making new thing more tedious to use (making it bigger 4nR, or making it a PITA to get) to make old thing seem appealing' but either way it's a no

(even if 'new thing' here is something we've always really had, between detect's revolver, sawing off mosins/shotguns, or the goodie-section pistols when those were a thing)

**Comment by Plum-but-gayer** [comment]
Torn on this because I love love love laser weapons becoming more viable, but also I recently got into using paxil's big yellow gun case which has the gimmick where you can fit a bulky weapon in it and this kinda defeats the purpose. Laser weapons definitely needed a buff though, right now you can get a wespe with 25 shot mags and reload that shit on the fly, I'm pretty sure that's more shots than a laser rifle has anyway? 

But also I'm thinking about some of our more powergamery players and looking at the screenshot MrMelbert posted and it's making me nervous. idk if this is the solution for energy weapons

**Comment by TheSS13Melon** [comment]
> Torn on this because I love love love laser weapons becoming more viable, but also I recently got into using paxil's big yellow gun case which has the gimmick where you can fit a bulky weapon in it and this kinda defeats the purpose. Laser weapons definitely needed a buff though, right now you can get a wespe with 25 shot mags and reload that shit on the fly, I'm pretty sure that's more shots than a laser rifle has anyway?
> 
> But also I'm thinking about some of our more powergamery players and looking at the screenshot MrMelbert posted and it's making me nervous. idk if this is the solution for energy weapons

Oh no, the powergamey players made it plainly obvious they're powergaming. What will we do

**Comment by Hatterhat** [comment]
> Those Who Do Not Learn History Are Doomed To Repeat It
> 
> image image

INSULS HOPCURITY THE CERTIFIED CLASSIC

**Comment by RatFromTheJungle** [comment]
> Those Who Do Not Learn History Are Doomed To Repeat It
> 
> image image

you should get on that 
image

**Comment by HoodMilk** [comment]
Not as crazy as people are making it out to be. After an hour into the round you are getting advanced eguns and mcrs which recharge on their own, which make carrying a load of guns less needed.

You can already have a load of ballistics in your bag, so who really cares?

**Comment by chemistrymain2** [comment]
considering laser weapons are lower damage than ballistics, this is fine

i'd much rather our ballistics were big, bulky, and high powered and our energy guns were simply easier to lug around and carry multiple of

**Comment by Plum-but-gayer** [comment]
> > Torn on this because I love love love laser weapons becoming more viable, but also I recently got into using paxil's big yellow gun case which has the gimmick where you can fit a bulky weapon in it and this kinda defeats the purpose. Laser weapons definitely needed a buff though, right now you can get a wespe with 25 shot mags and reload that shit on the fly, I'm pretty sure that's more shots than a laser rifle has anyway?
> > But also I'm thinking about some of our more powergamery players and looking at the screenshot MrMelbert posted and it's making me nervous. idk if this is the solution for energy weapons
> 
> Oh no, the powergamey players made it plainly obvious they're powergaming. What will we do

your words cut me like a knife. I'm crying now. :(

**Comment by RegJackson** [comment]
> Torn on this because I love love love laser weapons becoming more viable, but also I recently got into using paxil's big yellow gun case which has the gimmick where you can fit a bulky weapon in it and this kinda defeats the purpose. Laser weapons definitely needed a buff though, right now you can get a wespe with 25 shot mags and reload that shit on the fly, I'm pretty sure that's more shots than a laser rifle has anyway?
> 
> But also I'm thinking about some of our more powergamery players and looking at the screenshot MrMelbert posted and it's making me nervous. idk if this is the solution for energy weapons

realistically I don't think there's much practical difference between a guy carrying a wespe with an extendo and 3 spare extended mags, and a guy carrying 4 laser guns beyond a shittier wound type (pen-wounds last forever if you dont treat them) window penetration, damage numbers (.35 sol does more damage than stock lasers and only loses out by 5 to hellfires which have like 10 shots per gun), ammo cap (the guy with the wespe has 72 more shots) and a different damage type.

Also you should probably do something with thermal pistols while you're at it because they were the only Normal sized energy gun and that was part of their balancing point for having such shitty capacity. Maybe just pump it to 10 shots.

**Comment by lowpowermia** [comment]
god energy guns r really balanced for 100 hp carbons vs 140 hp..

**Comment by theselfish** [comment]
> > Those Who Do Not Learn History Are Doomed To Repeat It
> > image image
> 
> you should get on that image

Wow you sure showed them...Showing images of Skyrat guns.

**Comment by Stalkeros2** [comment]
Those are /tg/ not-Mosins.

**Comment by W4rd3nn** [comment]
the solution here is to make lasers the only viable option :3

**Comment by RatFromTheJungle** [comment]
> > > Those Who Do Not Learn History Are Doomed To Repeat It
> > > image image
> > 
> > 
> > you should get on that image
> 
> Wow you sure showed them...Showing images of Skyrat guns.

you should really check out TG code, namely, 'russian surplus crate'

**Comment by projectkepler-ru** [comment]
Don't the mosin have chances to jam and aren't they also hilariously inaccurate when sawn off? a better example would probably be the sawn off shotgun which doesn't lose accuracy and can still deal a fuck ton of damage

**Comment by chemistrymain2** [comment]
> Don't the mosin have chances to jam and aren't they also hilariously inaccurate when sawn off? a better example would probably be the sawn off shotgun which doesn't lose accuracy and can still deal a fuck ton of damage

technically if the mosin jams you just pull another mosin and keep firing

**Comment by Gandalf2k15** [comment]
nyet, tg did this for a reason
